Assets/Scripts/LinesCommandManager.cs

```diff
@@ -0,0 +1,130 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class LinesCommandManager : MonoBehaviour {
+
+    public static LinesCommandManager instance;
+
+    //lines
+    public GameObject linesGirl;
+    public GameObject linesGhost;
+
+    //text UI in the lines
+    public GameObject textGirl;
+    public GameObject textGhost;
+
+    private List<LinesCommand> commands = new List<LinesCommand>();
+    private int nowIndex = 0;   //current lines index
+    public bool isDialogueMode = true; //display mode flag
+
+    //cache transform
+    [SerializeField]
+    public List<string> startObj = new List<string>();
+    public Dictionary<string, Transform> allObj = new Dictionary<string, Transform>();
+
+    void Awake()
+    {
+        instance = this;
+    }
+
+    IEnumerator WaitAndPrint(float waitTime)
+    {
+
+        yield return new WaitForSeconds(waitTime);
+        
+        Next();
+    }
+
+    public void LineContainer(float waitTime)
+    {
+
+        StartCoroutine(WaitAndPrint(waitTime));
+    }
+
+
+    // Use this for initialization
+    void Start () {
+        isDialogueMode = true;
+        //set up game lines environment.
+        for (int i=0; i<startObj.Capacity; i++)
+        {
+            allObj.Add(startObj[i], GameObject.Find(startObj[i]).transform);
+        }
+        
+        commands.Add(new CommandDialogue("where is my brother?", textGirl, linesGirl));
```

Comment:
Just a comment, not a request of change:
To support multi-language, it's better to make them in a file, and add some information to tell the code it's language.